README.md

```diff
@@ -116,26 +116,48 @@ To work with a local (developer) installation of the Cortex Profiles SDK see [de
 examples:
 
-* [Using Local Cortex Clients](./local-clients/README.md)
-* [Join Two Connections](./join-connections/README.md)
-* [Refresh a DataSource](./datasource-refresh/README.md)
-* [Build Profiles](./build-profiles/README.md)
-* [Streaming to a Data Source](./datasource-streaming/README.md)
-* [Using a CData Connection](./cdata-connection/README.md)
-* [Reading From BigQuery](./bigquery-connection/README.md)
-* Profile Caching and retrieval
-  * [Caching Profiles](./cache-profile/README.md)
-  * [Profiles Daemon for Realtime Query](./profiles-daemon/README.md) <!-- Note: we should specify this and the previous example are used together for a larger example/use case-->
-* [KPI Queries](./kpi-queries/README.md)
-* [Filter and Aggregate Query examples](./filter-queries/README.md)
+| Sequence | Example                                                               | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+|----------|-----------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1        | [ Using Local Cortex Clients ]( ./local-clients/README.md )           | This example is intended to introduce working with the Cortex Profiles SDK in local development environment.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Couple of points, just my opinion:
1) I think I see what you mean about this table being a bit "hard to maintain" - like there's just a lot of words to read through and its partially (or entirely) copied from the examples. Would be nice to take a pass and skinny up the descriptions to make it a bit easier/quicker for visitors of the README. This table doesn't need full sentences, but its good that the descriptions here match the full sentence description from the individual READMEs.
2) Another option to quickly highlight properties of the examples would be to add another column like "Skill Type"   or catch-all "Tags". Not saying we need to do this, but if there is anything else you want to highlight